src/dishka/exceptions.py

```diff
@@ -1,134 +0,0 @@
-from collections.abc import Sequence
```

Comment:
If we remove this file, we should reexport everything moved in `__init__` not to break compatibility